project/home/templates/home/home_page.html

```diff
@@ -9,5 +9,12 @@
 <p>Welcome to Wagtail Social.</p>
 
 <div class="intro">{{ page.intro }}</div>
+{% for invite_friend in invite_friends %}
+    {% with invite_friend=invite_friend.specific %}
+        <div class="float-end">
+            <h2><a href="{% pageurl invite_friend %}">{{ invite_friend.title }}</a></h2>
```

Comment:
Slightly more explicit:

```suggestion
            <h2><a href="{% pageurl invite_friend %}">{{ invite_friends_page.title }}</a></h2>
```